source/languageHandler.py

```diff
@@ -201,37 +209,47 @@ def setLocale(localeName):
 		raise ValueError('unknown locale: %s' % localename)
 	ValueError: unknown locale: en-GB
 	'''
-	# Try setting Python's locale to lang
-	localeChanged = False
+	originalLocaleName = localeName
+	# Try setting Python's locale to localeName
 	try:
 		locale.setlocale(locale.LC_ALL, localeName)
 		locale.getlocale()
-		localeChanged = True
+		return

```

Comment:
I think we should have some thing in the debug log with the exact language code that successfully gets set